changelog.json

```diff
@@ -1,5 +1,10 @@
 {
   "releases": {
+    "2.0.4-beta1": [
+      "[Fixed] Refresh for Enterprise repositories did not handle API error querying branches - #7713",
+      "[Fixed] Missing \"Discard all changes\" context menu in Changes header - #7696",
+      "[Fixed] \"Select all\" keyboard shortcut not firing on Windows - #7759"
```

Comment:
This release also adds a `Discard all changes` shortcut and modifies the `Discard Repo` shortcut https://github.com/desktop/desktop/pull/7650

Reply:
Not just yet - this branch is built upon the latest release tag, so PRs merged into `development` won't be available just yet. We're planning our first 2.1 beta for next week some time, and this will definitely get a shout then!

Reply:
Ahh if only the PR screen could show the number of commits out of date a PR was....